### PR TITLE
Drop support for old versions

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is a BC break.

## Changelog

```markdown
### Removed
- Dropped support for old symfony (2-3) versions
- Dropped support for twig 1.x
- Dropped support for PHP 7.2 and lower
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,20 +45,13 @@ matrix:
     - php: '7.1'
```
Comment (OskarStark) [inline .travis.yml]: Please create a dev-kit PR for this, otherwise it gets overridden

Comment (core23) [reply .travis.yml]: Done

File: composer.json [hunk 2]
```diff
@@ -26,24 +26,24 @@
...
+        "symfony/http-kernel": "^4.3",
+        "symfony/options-resolver": "^4.3",
+        "symfony/twig-bundle": "^4.3",
         "twig/twig": "^1.34 || ^2.0"
     },
     "require-dev": {
         "knplabs/knp-menu-bundle": "^2.0",
-        "symfony/debug": "^2.8 || ^3.2 || ^4.0",
+        "symfony/debug": "^4.3",
         "symfony/phpunit-bridge": "^4.2",
```
Comment (greg0ire) [inline composer.json]: I don't really know if that will fix the issue, but maybe try bumping this to the latest version to make the prefer-lowest build pass.

**Comment by core23** [comment]
Should we drop twig 1 too? Not sure about this

File: composer.json [hunk 3]
```diff
@@ -20,30 +20,30 @@
...
+        "symfony/console": "^4.3",
+        "symfony/dependency-injection": "^4.3",
+        "symfony/event-dispatcher": "^4.3",
+        "symfony/form": "^4.3",
+        "symfony/framework-bundle": "^4.3",
+        "symfony/http-foundation": "^4.3",
+        "symfony/http-kernel": "^4.3",
+        "symfony/options-resolver": "^4.3",
+        "symfony/twig-bundle": "^4.3",
+        "twig/twig": "^2.1"
```
Comment (OskarStark) [inline composer.json]: Why not 2.0 ?

Comment (core23) [reply composer.json]: Changed this to 2.11 (latest minor release)

**Comment by OskarStark** [comment]
Can you please check travis?

**Comment by core23** [comment]
Depends on https://github.com/sonata-project/dev-kit/pull/502

**Comment by core23** [comment]
I would create a new PR with some cleanup afterwards

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**